samples/showcase/on-premises/Core_7/Store.ECommerce.Core/Startup.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using Alexinea.Autofac.Extensions.DependencyInjection;
+using Autofac;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NServiceBus;
+
+namespace Store.ECommerce.Core
+{
+    public class Startup
+    {
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+        public IServiceProvider ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc();
+            services.AddSignalR();
+            var builder = new ContainerBuilder();
+
+            builder.Populate(services);
+
+            IEndpointInstance endpoint = null;
+            builder.Register(c => endpoint)
+                .As<IEndpointInstance>()
+                .SingleInstance();
+
+            var container = builder.Build();
+
+            var endpointConfiguration = new EndpointConfiguration("Store.ECommerce");
+            endpointConfiguration.PurgeOnStartup(true);
+            endpointConfiguration.ApplyCommonConfiguration(transport =>
+            {
+                var routing = transport.Routing();
+                routing.RouteToEndpoint(typeof(Store.Messages.Commands.SubmitOrder).Assembly, "Store.Messages.Commands", "Store.Sales");
+            });
+            endpointConfiguration.UseContainer<AutofacBuilder>(
+                customizations: customizations =>
+                {
+                    customizations.ExistingLifetimeScope(container);
+                });
+
+            endpoint = Endpoint.Start(endpointConfiguration).GetAwaiter().GetResult();
+
+            return new AutofacServiceProvider(container);
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        {
+            if (env.IsDevelopment())
```

Comment:
Add shutdown similar to https://github.com/Particular/docs.particular.net/pull/3797